src/Nest/Document/Multiple/Bulk/BulkResponseItem/BulkResponseItemBase.cs

```diff
@@ -34,6 +34,14 @@ public abstract class BulkResponseItemBase
 		[DataMember(Name = "_primary_term")]
 		public long PrimaryTerm { get; internal set; }
 
+		[DataMember(Name = "get")]
+		internal LazyDocument Get { get; set; }
+
+		/// <summary>
+		/// Deserialize the <see cref="Get"/> property as a GetResponse<TDocument> type, where TDocument is the document type.
+		/// </summary>
+		public GetResponse<TDocument> GetResponse<TDocument>() where TDocument : class => Get?.AsUsingRequestResponseSerializer<GetResponse<TDocument>>();
```

Comment:
LGTM, but I think there should also be an integration test for this.

Reply:
Added a unit test